src/packages/core/workspace/entity-detail/entity-detail-workspace-base.ts

```diff
@@ -153,9 +152,7 @@ export abstract class UmbEntityDetailWorkspaceContextBase<
 	 * @memberof UmbEntityWorkspaceContextBase
 	 */
 	protected _checkWillNavigateAway(newUrl: string) {
-		const workspaceBasePath = UMB_WORKSPACE_PATH_PATTERN.generateLocal({ entityType: this.getEntityType() });
-		const currentWorkspacePathIdentifier = '/' + workspaceBasePath + '/' + this.routes.getActiveLocalPath();
-		return !newUrl.includes(currentWorkspacePathIdentifier);
+		return !newUrl.includes(this.routes.getActiveLocalPath());
```

Comment:
I'm not sure if this could give a problem if you had the same workspace with the same entity opened twice. Not a very likely case, but a case we should support. Lets talk about it, cause It just a thought.